ensime_shared/spec/features/ensime_harness.py

```diff
@@ -1,23 +0,0 @@
-class TestVimBuffer:
-    def __init__(self):
-        self.name = __file__
-
-class TestVimWindow:
-    def __init__(self):
-        self.cursor = [0, 0]
-
-class TestVimCurrent:
-    def __init__(self):
-        self.window = TestVimWindow()
-        self.buffer = TestVimBuffer()
-
-class TestVim:
-    def __init__(self):
-        self.current = TestVimCurrent()
-
-    def command(self, what):
-        return None
-
-    def eval(self, what):
-        return "/tmp"
```

Comment:
These were unused at this point, and I think they're most likely to belong in unit tests instead of integration. Stay tuned for some thoughts/work I have toward that.